test/unit/resources/package_test.rb

```diff
@@ -152,6 +152,15 @@
     _(resource.info).must_equal pkg
   end
 
+  # alpine
+  it 'verify Alpine package processing' do
+    resource = MockLoader.new(:alpine).load_resource('package', 'git')
+    pkg = { name: 'git', installed: true, version: '2.15.0-r1', type: 'pkg' }
+    _(resource.installed?).must_equal true
+    _(resource.version).must_equal '2.15.0-r1'
+    _(resource.info).must_equal pkg
+  end
+
```

Comment:
Great work, you need to add the command output for `apk info -vv --no-network | grep git` in https://github.com/inspec/inspec/tree/master/test/unit/mock/cmd and the command registration similar to https://github.com/inspec/inspec/blob/master/test/helper.rb#L244

Reply:
Thank you, I was wondering how/where to stub and mock that data.